src/main/kotlin/com/toasttab/pulseman/state/PulsarSettings.kt

```diff
@@ -63,49 +71,63 @@ class PulsarSettings(
     private val dependencyJarManagement =
         JarManagement(appState.dependencyJars, null, setUserFeedback, onChange)
 
-    val showDiscover = mutableStateOf(false)
-    val showJarManagement = mutableStateOf(false)
-    val showAuthSettings = mutableStateOf(false)
-    val showPropertySettings = mutableStateOf(false)
+    private val showDiscover = mutableStateOf(false)
+    private val showJarManagement = mutableStateOf(false)
+    private val showAuthSettings = mutableStateOf(false)
+    private val showPropertySettings = mutableStateOf(false)
 
-    fun onShowDiscoverChange(newValue: Boolean) {
-        showDiscover.value = newValue
-    }
-
-    fun onShowJarManagementChange(newValue: Boolean) {
-        showJarManagement.value = newValue
-    }
-
-    fun onShowAuthSettingsChange(newValue: Boolean) {
-        showAuthSettings.value = newValue
-    }
-
-    fun onShowPropertySettingsChange(newValue: Boolean) {
-        showPropertySettings.value = newValue
-    }
-
-    fun onTopicChange(newValue: String) {
+    private fun onTopicChange(newValue: String) {
         topic.value = newValue
         onChange()
     }
 
-    fun onServiceUrlChange(newValue: String) {
+    private fun onServiceUrlChange(newValue: String) {
         serviceUrl.value = newValue
         onChange()
     }
 
-    val jarManagementTabs = JarManagementTabs(
+    private val jarManagementTabs = JarManagementTabs(
         listOf(
-            Pair("Message", pulsarJarManagement),
-            Pair("Auth", authJarManagement),
-            Pair("Other", dependencyJarManagement)
+            Pair(MESSAGE, pulsarJarManagement),
+            Pair(AUTH, authJarManagement),
+            Pair(OTHER, dependencyJarManagement)
         )
     )
 
-    companion object {
-        val popupState = DialogState().apply {
-            size = WindowSize(750.dp, 600.dp)
+    @ExperimentalFoundationApi
+    @ExperimentalComposeUiApi
```

Comment:
Is it still experimental now that's `1.0.0`?

Reply:
I went through them all and tried to remove them and some are still needed, did it again and somehow I missed two I could remove thanks